packages/vscode-extension/src/extension/main.ts

```diff
@@ -47,73 +47,86 @@ export function activate(context: vscode.ExtensionContext): void {
 }
 
 /**
- * Listen for file open events, and prompt if we can create a .pliplugin folder
+ * Listen for changes on file activation, and prompt if we can create a .pliplugin folder
  * @returns Disposable listener
  */
+let shouldShowInfoMessage = true;
 function registerOnDidOpenTextDocListener(
   telemetryReporter: TelemetryReporter | undefined,
 ) {
-  const listener = vscode.workspace.onDidOpenTextDocument(async (document) => {
-    // settle on the 1st workspace folder available
-    // TODO @montymxb May 15th, 2025: Support configs across multiple workspace folders
-    const workspaceFolder = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath;
-    if (!workspaceFolder) {
-      return;
-    }
+  const listener = vscode.window.onDidChangeActiveTextEditor(
+    async (document) => {
```

Comment:
We should call this `editor`, `textEditor`, or something in that vein; but it's definitely not a document. I was wondering when I saw `document.document` below 🤔 .